### PR TITLE
Robust module in commercial.js

### PR DESCRIPTION
**Pull request by Calanthe**
The robust module was added to the commercial.js. Now for example, third party tags throwing an exception should not stop displaying ads.

### CODE REVIEW

File: static/src/javascripts/bootstraps/commercial.js [hunk 1]
```diff
@@ -31,16 +33,16 @@ define([
...
+                robust('cm-articleAsideAdverts',           function () { articleAsideAdverts.init(); });
+                robust('cm-articleBodyAdverts',            function () { articleBodyAdverts.init(); });
+                robust('cm-sliceAdverts',                  function () { sliceAdverts.init(); });
+                robust('cm-frontCommercialComponents',     function () { frontCommercialComponents.init(); });
+                robust('cm-badges',                        function () { badges.init(); });
+                robust('cm-dfp',                           function () { dfp.init(); });
             }
 
-            mediator.emit('page:commercial:ready');
+            robust('cm-ready',      function () { mediator.emit('page:commercial:ready'); });
```
Comment (ironsidevsquincy) [inline static/src/javascripts/bootstraps/commercial.js]: lots of space after comma

File: static/src/javascripts/core.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ require([
     'common/utils/storage',
     'common/utils/template',
     'common/utils/url',
+    'common/utils/robust',
```
Comment (ironsidevsquincy) [inline static/src/javascripts/core.js]: alphabetical ordering

File: static/src/javascripts/bootstraps/commercial.js [hunk 3]
```diff
@@ -31,16 +33,16 @@ define([
...
-                frontCommercialComponents.init();
-                badges.init();
-                dfp.init();
+                robust('cm-thirdPartyTags',                function () { thirdPartyTags.init(); });
+                robust('cm-articleAsideAdverts',           function () { articleAsideAdverts.init(); });
+                robust('cm-articleBodyAdverts',            function () { articleBodyAdverts.init(); });
+                robust('cm-sliceAdverts',                  function () { sliceAdverts.init(); });
+                robust('cm-frontCommercialComponents',     function () { frontCommercialComponents.init(); });
+                robust('cm-badges',                        function () { badges.init(); });
+                robust('cm-dfp',                           function () { dfp.init(); });
```
Comment (ironsidevsquincy) [inline static/src/javascripts/bootstraps/commercial.js]: think you can take 4 or so spaces away after the commas

**Comment by phamann** [comment]
:+1:

**Pull request merged**